### PR TITLE
Use namespace for helm install name

### PR DESCRIPTION
**Pull request by alexjh**

### CODE REVIEW

File: make/upgrade [hunk 1]
```diff
@@ -15,8 +15,7 @@ cd "${GIT_ROOT}"
 . make/include/secrets
 
 # Argument? (**)
-RELEASE=$(helm list --date --reverse --max 1 --namespace "${NAMESPACE}" -q | tail -n 1)
-echo Upgrading ${NAMESPACE} release \"${RELEASE}\" ...
+echo Upgrading ${NAMESPACE} release \"${NAMESPACE}\" ...
```
Comment (mook-as) [inline make/upgrade]: This would be clearer, I think, if we do a `RELEASE="${NAMESPACE}" (with appropriate comment) instead?

File: make/upgrade [hunk 2]
```diff
@@ -38,15 +37,16 @@ source "${NETWORK_ENV}"
 
 UAA_CA_CERT="$(get_secret "${UAA_NAMESPACE}" "uaa" "INTERNAL_CA_CERT")"
 
-helm upgrade ${RELEASE} output/helm \
-    --namespace ${NAMESPACE} \
+helm upgrade "${NAMESPACE}" output/helm \
+    --name "${NAMESPACE}" \
```
Comment (mook-as) [inline make/upgrade]: Does `helm upgrade` take a `--name`?

File: make/upgrade (hunk 1 above)
Comment (alexjh) [reply make/upgrade]: :+1: Fixed and rebased to simplify the diff.

File: make/upgrade (hunk 2 above)
Comment (alexjh) [reply make/upgrade]: Oops, no, it doesn't. Fixed in the rebased branch.

**Pull request merged**